db/dcrsqlite/retrieveallpoolvalandsize_test.go

```diff
@@ -0,0 +1,23 @@
+package dcrsqlite
+
+import (
+	"testing"
+
+	"github.com/decred/dcrdata/testutil"
+)
+
+func TestEmptyDBRetrieveAllPoolValAndSize(t *testing.T) {
+	testutil.BindCurrentTestSetup(t)
+	db := InitTestDB(DBPathForTest())
+	testEmptyDBRetrieveAllPoolValAndSize(db)
+}
+
+func testEmptyDBRetrieveAllPoolValAndSize(db *DB) {
+	result, err := db.RetrieveAllPoolValAndSize()
+	if err != nil {
+		testutil.ReportTestFailed(
+			"RetrieveAllPoolValAndSize() failed: default result expected:",
+			err)
+	}
+	checkChartsDataIsDefault("RetrieveAllPoolValAndSize", result)
```

Comment:
So instead of all that other code, you can just check that the pointer is non-nil, and the data is default with `DeepEqual` as I said.  Or you could use something like https://github.com/google/go-cmp